lib/src/update_status.dart

```diff
@@ -11,42 +11,28 @@ class UpdateStatus {
 
   final String? status;
   final int? updateId;
-  final UpdateType? type;
-  final double? duration;
+  final String? type;
+  final String? duration;
   final DateTime? enqueuedAt;
```

Comment:
I realize that I also forgot to add the `details` field to the task too.
I'm not sure how complicated it is in `Dart` to add a field without a preset. If not, it could be good, and if not, too bad.

Reply:
Good catch, I'll implement that :)